### DISCUSSION TITLE
'woff2_compress' command not working!

### DISCUSSION THREAD

**Discussion by IzumiSenaSora** [termux/termux-packages#18541] [Q&A] [answered]
Is this package available? I installed it `pkg install woff2` but this command not working... Is it different from `https://github.com/google/woff2` ?

**Comment by sylirre** [comment]
~~What output do you have when running `pkg install woff2`?~~

Edit: `woff2` package contains only libraries. This is why command doesn't work.

**Reply by sylirre** [reply, marked as answer]
Will be fixed in version `1.0.2-3`: https://github.com/termux/termux-packages/commit/a0cde7c314005956e6147b428093acfd00bb5f89
